src/main/java/org/example/AccountInputService.java

```diff
@@ -0,0 +1,65 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class AccountInputService {
+    Scanner sc = new Scanner(System.in);
+    public AccountType getAccountType(){
+        System.out.print("""
+                1. Basic Banking Account
+                 Account can be opened with an initial/minimum deposit of Rs.1, 000.
+                2. Saving Account
+                 Account can be opened with an initial/minimum deposit of Rs.5,000
+                3. Current Account
+                 No Minimum Balance
+                Select Account Type:\s""");
+        int Type = sc.nextInt();
+        for (AccountType accountType : AccountType.values()) {
+            if (accountType.getValue() == Type) {
```

Comment:
its same thing, you are looping and having if condition,
it could be better than previous if conditions, but can you completely eradicate conditional thing, just like you have done in gender thing